cli/dcoscli/marathon/main.py

```diff
@@ -980,6 +1062,37 @@ def _ensure_pods_support(marathon_client):
             raise DCOSException(msg)
 
 
+def _enhance_row_with_overdue_information(rows, queued_apps):
+    """Calculates if configured `backoff` duration for this
+    app or pod definition was exceeded. In that case this application
+    is marked as `overdue`.
+    If the app or pod inside this row should be
+
+    :param rows: list of rows
+    :type rows: []
+    :param queued_apps: list of
+    :type queued_apps: []
+    :returns: true if pod is overdue, false otherwise
+    :rtype: bool
+    """
+
+    for row in rows:
+        queued_app = next(
+            (app for app in queued_apps
+             if row.get('id') == marathon.get_app_or_pod_id(app)),
+            None)
+        if queued_app:
+            overdue = queued_app.get('delay', {}).get('overdue', False)
```

Comment:
You can simplify this as `row['overdue'] = queued_app.get('delay', {}).get('overdue', False)`.